cdci_osa_plugin/osa_common_pars.py

```diff
@@ -120,14 +147,8 @@ def osa_common_instr_query():
 
     radius = Angle(value=5.0, units='deg', name='radius')
 
-    osa_version = OSAVersion(name_format='str', name='osa_version', value='OSA11.1')
-    if  os.environ.get('DISPATCHER_MOCK_KB', 'no') == 'yes' or 'cdciweb01' in socket.gethostname():
-        osa_version._allowed_values = [
-            'OSA10.2', 'OSA11.0', 'OSA11.1']  # this really only for test
-    else:
-        osa_version._allowed_values = get_osa_versions()
-        # can not really naturally select here by token roles
-
+    osa_version = OSAVersion(name='osa_version', value='OSA11.1', allowed_base_osa_version_values=["OSA10.2", "OSA11.1"])
```

Comment:
Is `OSA11.0` no longer accepted? I recall it still was but with a warning.

Reply:
it was not not a warning, it was not accepted. Just in different place, in role checking.
I just moved that validation, since it's not role-specific at this time.